app/models/match_result.rb

```diff
@@ -0,0 +1,41 @@
+# == Schema Information
+#
+# Table name: match_results
+#
+#  id               :integer          not null, primary key
+#  match            :integer          (points to a Match Object)
+#  winner_advances  :boolean
+#
+
+class MatchResult < ApplicationRecord
+  belongs_to :match
+
```

Comment:
Since you will always want the relevant match of a MatchResult when working with that MatchResult, you should [eagerly load](http://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations) the `:match` in the MatchResult [`default_scope`](http://guides.rubyonrails.org/active_record_querying.html#applying-a-default-scope).